src/main/java/com/google/sps/data/Search.java

```diff
@@ -80,6 +100,16 @@ public boolean eventInCategory(String category) {
     || category.equals(this.category);
   }
 
+  public int countMatchingTags(List<String> tags) {
```

Comment:
What is the reason to return -1 for `this.tags.size() == 0`?

Reply:
To distinguish between empty search and empty tags for an events. The event should be added for the list if the tag search is empty and shouldn't when the event's list of tags is empty.